lib/main.dart

```diff
@@ -71,9 +73,9 @@ void main() async {
             RepositoryProvider(
               create: (context) => GetAllLiquidityInfoUseCase(_subGraphRepo),
             ),
-            // RepositoryProvider(
-            //   create: (context) => (_subGraphRepo),
-            // ),
+            RepositoryProvider(
+              create: (context) => _trackingRepository,
+            ),
```

Comment:
Only instantiate the repository outside of `create` if you need to pass it in multiple places, otherwise instantiate it directly in `create`.